spec/formatting.md

```diff
@@ -197,18 +197,28 @@ the following steps are taken:
 
 1. If the _expression_ includes an _operand_, resolve its value.
    If this fails, use a _fallback value_ for the _expression_.
-2. Based on the _function_ starting sigil and _name_,
-   find the appropriate function implementation from the _function registry_.
-   If the registry does not define an implementation for this _name_,
-   emit an Unknown Function error
+2. _Resolve the name_ of the _function_ and, based on the starting sigil,
+   find the appropriate function implementation to call.
+   If the implementation cannot find the function, emit an Unknown Function error
    and use a _fallback value_ for the _expression_.
-3. Resolve the _option_ values to a mapping of string identifiers to values.
+
+   An implementation MAY emit an Unknown Function error if the implementation    
+   does not support the _namespace_ of the _name_
+   or the _namespace_ does not support the _name-part_ named _function_.
```

Comment:
This sentence does not add anything to the previous one. If either of the conditions here are true, then it's also true that the implementation "cannot find the function", and will therefore emit the error.

```suggestion
```